code/drasil-printers/lib/Language/Drasil/Markdown/CreateMd.hs

```diff
@@ -44,13 +44,20 @@ commandLine = text $ "In your terminal command line, enter the same directory as
 -- | Helper for giving instructions on how to run the program.
 runInstDoc :: Doc
 runInstDoc = text "How to Run the Program:" <> contSep <>
-    commandLine <> contSep <> bkQuote3 <> contSep <> text "make run RUNARGS=input.txt" <> contSep <> bkQuote3
+    commandLine <> contSep <> bkQuote3 <> contSep <> text "make run RUNARGS=input.txt" 
+      <> contSep <> bkQuote3
 
 -- | Helper for giving instructions on how to build the program.
 makeInstDoc :: Doc
 makeInstDoc = text "How to Build the Program:" <> contSep <> commandLine <> contSep <>
     bkQuote3 <> contSep <> text "make build" <> contSep <> bkQuote3
 
+-- | Helper for giving instuctions and Input and Output files.
+inOutFile :: String -> Doc
+inOutFile name = text "How to Change Input:" <> contSep <> text name <+> 
+    text "will take the inputs from input.txt and write the outputs to output.txt." 
+    <> contSep <> text "Inputs can be changed by editing input.txt."
```

Comment:
Is there a way to pull "input.txt" and "output.txt" from the filenames themselves? Do we actually specify the file names anywhere or is this hardcoded when we actually generate them?

Reply:
I agree that those should be parameters to `inOutFile` even if they end up being hard-coded constants elsewhere in the code.